streamfx-force-frontend.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
This patch is necessary to enable the front-end. The build system [checks for the presence](https://github.com/Xaymar/obs-StreamFX/blob/master/CMakeLists.txt#L991) of `${libobs_SOURCE_DIR}/cmake/obs-frontend-api/obs-frontend-apiConfig.cmake`, which apparently isn't shipped in the Flatpak of OBS.

@GeorgesStavracas do you know if something can be done about it in OBS?

Reply:
This is more of an artifact of two conflicting PRs sent to upstream, one which only added the header file and the other which turned obs-frontend-api into its own module separate from libobs/obs. Unfortunately the former was merged and the latter was rejected.

There was/is also a separate PR that completely redesigns the CMake file for obs-studio, which is what I am waiting for before I redesign the CMake file for StreamFX again. I'm currently failing at finding the link for the CMake rework PR.